tests/api_connexion/endpoints/test_task_instance_endpoint.py

```diff
@@ -873,6 +873,20 @@ def test_should_raise_403_forbidden(self):
         )
         assert response.status_code == 403
 
+    @pytest.mark.parametrize(
+        "payload, expected",
+        [
+            ({}, "POST Body must not be empty"),
+        ],
+    )
```

Comment:
Do you plan to add more cases? A single-case paramtrisation is not useful, from what I can tell.

Reply:
It's meant to be a single test-case, when POST Body is empty. I can remove the parametrize if required?

Reply:
Yes please do

Reply:
Removed LGTM -> merge.